sync/validation_queue.go

```diff
@@ -208,28 +209,32 @@ func (vq *blockQueue) addDependencies(jobID interface{}, blks []types.BlockID, f
 		return false, fmt.Errorf("job %s already exsits", jobID)
 	}
 
-	dependencys := make(map[types.Hash32]struct{})
+	dependencies := make(map[types.Hash32]struct{})
 	idsToPush := make([]types.Hash32, 0, len(blks))
 	for _, id := range blks {
 		bid := id.AsHash32()
 		if vq.inQueue(bid) {
 			vq.reverseDepMap[bid] = append(vq.reverseDepMap[bid], jobID)
-			vq.Debug("add block %v to %v pending map", id, jobID)
-			dependencys[bid] = struct{}{}
+			vq.With().Debug("adding already queued block to pending map",
+				log.BlockID(id.String()),
+				log.JobID(jobID))
+			dependencies[bid] = struct{}{}
 		} else {
 			//	check database
 			if _, err := vq.GetBlock(id); err != nil {
-				//unknown block add to queue
+				//add unknown block to queue
```

Comment:
I think we should always add a space after the `//` this is inconsistent throughout this code, up to you if you want to fix (if you tell your IDE you want a space there, it's just a matter of using its format tool = one click).

Reply:
the scope of this PR is fixing logs, not comments 😊 will fix the ones I see!